grid-proxy/pkg/types/types.go

```diff
@@ -258,14 +258,53 @@ type RentContractDetails struct {
 	NodeID uint `json:"nodeId"`
 }
 
+type ContractDetails interface{}
+
 type Contract struct {
-	ContractID uint              `json:"contractId"`
-	TwinID     uint              `json:"twinId"`
-	State      string            `json:"state"`
-	CreatedAt  uint              `json:"created_at"`
-	Type       string            `json:"type"`
-	Details    interface{}       `json:"details"`
-	Billing    []ContractBilling `json:"billing"`
+	ContractID uint            `json:"contract_id"`
+	TwinID     uint            `json:"twin_id"`
+	State      string          `json:"state"`
+	CreatedAt  uint            `json:"created_at"`
+	Type       string          `json:"type"`
+	Details    ContractDetails `json:"details"`
```

Comment:
i believe that the details is creating some unneeded hassle, maybe we should change its type??